### ISSUE TITLE
HWI issue with Trezor One and Sparrow (slow loading times)

### ISSUE THREAD

**Issue by bedwards110** [bitcoin-core/HWI#727]
I've had slow loading time issues with Trezor One and Sparrow on my desktop. I opened a ticket on Sparrow Github https://github.com/sparrowwallet/sparrow/issues/1315#issue-2181812555 and was told to open one here. My Trezor One takes anywhere from 5 to 10 minutes to show up on Sparrow once connected, then another 5 to 10 minutes to log in via PIN. I've tried this on three laptops and it connects within 10 seconds. Sparrow support says its an HWI issue. Here is the code cmd returned after running this code: C:\Users\Brandon\AppData\Roaming\Sparrow\hwi\hwi-2.4.04336763555989704409.tmp --debug enumerate

INFO:hwilib.devices.trezorlib.client:creating client instance for device: webusb:001:10
DEBUG:hwilib.devices.trezorlib.client:sending message: GetFeatures
DEBUG:hwilib.devices.trezorlib.client:received message: Features
DEBUG:hwilib.devices.trezorlib.client:sending message: Initialize
DEBUG:hwilib.devices.trezorlib.client:received message: Features
INFO:hwilib.devices.jadepy.jade_tcp:Connecting to tcp:127.0.0.1:30121
DEBUG:root:Failed to connect to Jade simulator at tcp:127.0.0.1:30121
DEBUG:root:timed out
[{"type": "trezor", "path": "webusb:001:10", "label": "Brandon Trezor", "model": "trezor_1", "needs_pin_sent": true, "needs_passphrase_sent": false, "error": "Could not open client or get fingerprint information: Trezor is locked. Unlock by using 'promptpin' and then 'sendpin'.", "code": -12}]

**Comment by brunoerg** [comment]
I didn't see how sparrow uses this, but it seems it's spending time trying to connect to Jade simulator and timing out. So, simulators are allowed.
```
INFO:hwilib.devices.jadepy.jade_tcp:Connecting to tcp:127.0.0.1:30121
DEBUG:root:Failed to connect to Jade simulator at tcp:127.0.0.1:30121
DEBUG:root:timed out
```

**Comment by achow101** [comment]
Should be resolved #729
